hdInsight-apache-spark-1-4-1/readme.md

```diff
@@ -0,0 +1,31 @@
+# Create HDInsight Linux cluster and run custom script action to install Apache Spark 1.4.1
+Creates HDInsight Linux cluster and run custom script action to install Apache Spark 1.4.1<br>
```

Comment:
Can you rename this template to hdInsight-apache-spark so that it sticks with the other hdinsight templates in the list and when a newer spark version is released a folder rename isn't required